package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "wolox-equalizer",
```

Comment:
Is this package only going to have the equalizer? Or you will add layouts and margins and paddings and so on?

Reply:
only equalizer @mcallegari10